hello-strongbox-gradle/README.md

```diff
@@ -0,0 +1,12 @@
+This is an example of how to use the Strongbox artifact repository manager with [Gradle](http://gradle.org/).
+
+# Before you start
+Make sure that your Strongbox instance is up and running. If you are new to Strongbox, please visit the [Installation](https://github.com/strongbox/strongbox#installation) page first.
+
+# Requirements
+You will need the following software installed on your machine to make this example working:
+* Java Development Kit (JDK) version 1.8.x or higher
+* [Gradle](http://gradle.org/) version 2.11 or higher
+
+# How to build from source code
+When you clone this example project sources, go into that directory and simply type `gradle build`. 
```

Comment:
Could you please further extend these explanations a bit to include the most basic things about Gradle that one should know (like I've done [here](https://github.com/strongbox/strongbox-examples/tree/master/hello-strongbox-maven))?

Also, could you please provide the output of the build as an example of how things should be invoked and how the successful output would look like?

Many thanks! :)